lib/utils.js

```diff
@@ -91,30 +91,44 @@ function getPageLinks(page, footIndex)
 
 function isInternalLink(link)
 {
-  return !url.parse(link.url || link).host
+  return !url.parse(typeof link.url === 'string' ? link.url : link).host
 }
 
-function prepareInternalLink(link, page)
+function prepareInternalLink(link, page, book)
 {
   var linkUrl = link.url
 
   // Resolve link path
   linkUrl = decodeURI(url.resolve(page.path, linkUrl))
+  // get linked page title & create md link
+  linkedPage = book.getPageByPath(linkUrl)
+
+  if(!linkedPage)
+    return book.log.warn('[printlinks] page '+linkUrl+' not found (linked from'+page.title+')')
+
+  linkUrl = `[${linkedPage.level} ${linkedPage.title}](${linkUrl})`
```

Comment:
because it's nice to be able to click on things in PDFs
i know, the plugin is about printouts, but the output is still a pdf